package/tests/test_server.py

```diff
@@ -289,3 +315,18 @@ def test_call_twice_with_different_port(self, mocked_process):
             target=server._call_viz, kwargs={"port": 8000}, daemon=True
         )
         assert mocked_process.call_count == 2
+
+    def test_check_viz_up(self, requests_mock):
+        """
+        Test the helper function which checks if HTTP GET status code is 200.
+        """
+        requests_mock.get(
+            "http://127.0.0.1:8000/", content=b"some output", status_code=200
+        )
+        assert server._check_viz_up(8000)  # pylint: disable=protected-access
+
+    def test_check_viz_up_invalid(self):
+        """
+        Test should catch the request connection error and returns false.
```

Comment:
```suggestion
        Test should catch the request connection error and returns False.
```
I'd rename the tests to be more explicit that one checks the status_code and one the connection error scenario. "invalid" is a bit vague.